WebApplication1/WebApplication1.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
Don't forget to delete this test app WebApplication1/WebApplication1.csproj

Reply:
I tought I did? I'll take a look